### ISSUE TITLE
No responding since 2.0.0

### ISSUE THREAD

**Issue by ray3730** [IUrreta/DatabaseEditor#229]
Really appreciate all the work and been enjoying this up to 1.8.3.
Howecer starting from 2.0.0 when I clicn "run" simply no response. Not sure anyone else encountered this.

Thanks

**Comment by ray3730** [comment]
also re downloaded python node js and git to git install the editor, used launch.sh to install all necessary file. But still no response, no log file created either. Quite weird as everything is perfect up to 1.8.3

Im using Win 10. Thanks in advance

**Comment by IUrreta** [comment]
Try launching with launch.bat in the launcher folder, to sse what then console says. if python says that everything is okay but then it closes quickly, delete node_modules folder (in the Database Editor folder) and try running again

**Comment by ray3730** [comment]
> Try launching with launch.bat in the launcher folder, to sse what then console says. if python says that everything is okay but then it closes quickly, delete node_modules folder (in the Database Editor folder) and try running again

Thanks for responding. Yeah I tried the launch.bat in the launcher folder and quickly screenshot the black window. Says everything is satisfied and the window quickly closed. Then I deleted node_modules folder in the database editor folder and hit run again. The node.js folder gets re-created, but still no response and no log file. Weird as in previous versions it runs all good. Thanks for looking into it. Cheers
<img width="1920" alt="2024-07-24 (10)" src="https://github.com/user-attachments/assets/af98f52d-91a6-4355-b32a-1a448b715951">

**Comment by IUrreta** [comment]
the console should say something just before closing very quickly. normally when this happens is that electron failed ot install (a dependency from the node_modules folder), and just by deleting the node_modules folder (or the electron folder inside the node_modules folder) and re-running it should work. Maybe you can screen-record to try to see if it says anything else before closing?

**Comment by ray3730** [comment]
> the console should say something just before closing very quickly. normally when this happens is that electron failed ot install (a dependency from the node_modules folder), and just by deleting the node_modules folder (or the electron folder inside the node_modules folder) and re-running it should work. Maybe you can screen-record to try to see if it says anything else before closing?

![electron](https://github.com/user-attachments/assets/9d823168-0e0a-482d-af45-0d59608bc59e)
That is correct. I managed to video record the electron error right before it closed. However when I deleted the node_modules folder and hit run.vbs. A new node_modules folder was created but still no editor showed up. And the same electron error occured. I'm using the latestr 22.5.1 version. Weird. Thanks for helping

**Comment by IUrreta** [comment]
Are you in china or asia? I have had multiple people from china having trouble downloading electron, seems to be a localization problem

**Comment by ray3730** [comment]
> Are you in china or asia? I have had multiple people from china having trouble downloading electron, seems to be a localization problem

Yes Im in China. And this time I tried the launch.bat in the launcher folder using admin privilege and seems there is an issue with npm/operation not permitted. I managed to find the log file. It's strange as I do have admin privilege and already changed the entire C drive security permission to full control for everyone. I also tried to force clean the npm cache twice. But still the issue persists.
[2024-07-24T09_39_03_448Z-debug-0.log](https://github.com/user-attachments/files/16360304/2024-07-24T09_39_03_448Z-debug-0.log)
